api/interfaces/pinia.DefineStoreOptionsInPlugin.md

```diff
@@ -6,84 +6,81 @@ sidebarDepth: 3
 
 [API Documentation](../index.md) / [pinia](../modules/pinia.md) / DefineStoreOptionsInPlugin
```

Comment:
Missing translation to "Documentación de la API"

Reply:
Fixed 🚀 Gracias por notarlo